de.metas.business/src/main/java-legacy/org/compiere/model/MOrder.java

```diff
@@ -1158,6 +1247,23 @@ public String prepareIt()
 			return IDocument.STATUS_Invalid;
 		}
 
+		// Credit Limit
+		if (isSOTrx())
+		{
+			final I_C_BPartner partner = InterfaceWrapperHelper.load(getC_BPartner_ID(),  I_C_BPartner.class);
```

Comment:
we already have a column `C_BPartner.SO_CreditLimit` which is used at different places..e.g. in `MOrder.prepareIt()`, some lines below this code.
i propose to remove C_BPartner.SO_CreditLimit`, unify and and migrate to and use the new datastructure everywhere. or are they about different stuff?

Reply:
Hm... I think you are right.
I was thinking to do it in a further step, but, does not makes sense.